packages/trader/scripts/WithdrawTokensFromDSProxy.ts

```diff
@@ -0,0 +1,89 @@
+// This script a withdrawal transaction that will pull founds out of a DSProxy contract. The wallet you run the script
+// from should be the owner (or have sufficient permissions).
+// To Execute execute the script as: truffle exec ./scripts/WithdrawTokensFromDSProxy.ts --network kovan_mnemonic --dsProxyAddress 0x... --tokenAddress 0x... --amount max -- recipientAddress 0x...
+// Note:
+// 1) if you do not provide the dsProxyAddress the script will try find one deployed at the unlocked wallet account.
+// 2) if you provide max for amount then the script will take all tokens. If you provide a spesific number, it is assumed
+// to be a string. No internal scalling is done on the number. 1 eth should be therefore represented as 1000000000000000000
```

Comment:
nit: maybe not worth it for this PR, but could be nice to grab the decimals of the token and use that to scale? Thoughts?

Reply:
You could. For now I thought it was more explicit and easier to reason about like this.